apps/dashboard/src/@/styles/globals.css

```diff
@@ -15,7 +15,7 @@
 
     /* bg - colorful */
     --primary: 221.21deg 83.19% 53.33%;
-    --destructive: 0 84.2% 90%;
+    --destructive: 0 84.2% 60.2%;
```

Comment:
why did this change?

Reply:
The previous color wasn't red (it was some shade of gray), I'm not sure if destructive-foreground and destructive got mixed up or what